presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -414,8 +310,196 @@ as a part of SQL query by appending name with the table.
+    )
+    WITH (
+      format = 'ORC',
+      partitioning = ARRAY['ds', 'country'],
+      format_version = '2'
+    )
+
+INSERT INTO
+^^^^^^^^^^^^
+
+Insert data into ``page_views`` table::
+
+    INSERT INTO iceberg.web.page_views VALUES(TIMESTAMP '2023-08-12 03:04:05.321', 1, 'https://example.com', current_date, 'country');
+
+CREATE TABLE AS SELECT
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Create a new table ``page_views_new`` from an existing table ``page_views``::
+
+    CREATE TABLE iceberg.web.page_views_new AS SELECT * FROM iceberg.web.page_views
+
+SELECT
+^^^^^^
+
+SELECT table operations are supported for Iceberg format version 1 & version 2 in the connector::
+
+    SELECT * FROM iceberg.web.page_views;
+
+    SELECT * FROM iceberg.web.page_views_v2;
+
+.. note::
+
+    Iceberg Tables with format version 2 where delete files are present are not filtered in ``SELECT``
+    operations in Presto.
+
+ALTER TABLE
+^^^^^^^^^^^^
+
+Alter table operations are supported in the connector::
+
+     ALTER TABLE iceberg.web.page_views ADD COLUMN zipcode VARCHAR;
+
+     ALTER TABLE iceberg.web.page_views RENAME COLUMN zipcode TO location;
+
+     ALTER TABLE iceberg.web.page_views DROP COLUMN location;
+
+DROP TABLE
+^^^^^^^^^^^
+
+Drop the table ``page_views``. This only drops the metadata
+& data for the table::
+
+    DROP TABLE iceberg.web.page_views
+
+DROP SCHEMA
+^^^^^^^^^^^^
+
+Drop a schema::
+
+    DROP SCHEMA iceberg.web
+
+Schema Evolution
+-----------------
+
+Iceberg and Presto Iceberg connector supports in-place table evolution, aka
+schema evolution, such as adding, dropping, and renaming columns. With schema
+evolution, users can evolve a table schema with SQL after enabling the Presto
+Iceberg connector.
+
+Example Queries
+^^^^^^^^^^^^^^^
+
+Let's create an Iceberg table named `ctas_nation`, created from the TPCH `nation`
+table. The table has four columns: `nationkey`, `name`, `regionkey`, and `comment`.
+
+.. code-block:: sql
+
+    USE iceberg.tpch;
+    CREATE TABLE IF NOT EXISTS ctas_nation AS (SELECT * FROM nation);
+    DESCRIBE ctas_nation;
+
+.. code-block:: text
+
+      Column   |  Type   | Extra | Comment
+    -----------+---------+-------+---------
+     nationkey | bigint  |       |
+     name      | varchar |       |
+     regionkey | bigint  |       |
+     comment   | varchar |       |
+    (4 rows)
+
+We can simply add a new column to the Iceberg table by using the `ALTER TABLE`
+statement. The following query adds a new column named `zipcode` to the table.
+
+.. code-block:: sql
+
+    ALTER TABLE ctas_nation ADD COLUMN zipcode VARCHAR;
+    DESCRIBE ctas_nation;
+
+.. code-block:: text
+
+      Column   |  Type   | Extra | Comment
+    -----------+---------+-------+---------
+     nationkey | bigint  |       |
+     name      | varchar |       |
+     regionkey | bigint  |       |
+     comment   | varchar |       |
+     zipcode   | varchar |       |
+    (5 rows)
+
+We can also rename the new column to `location`:
```

Comment:
to `location`. -> to another name, e.g. `location`

I feel just "location" would likely to be confused with the table property "location"